src/Tizen.NUI/src/public/BaseComponents/ViewEnum.cs

```diff
@@ -199,11 +199,9 @@ internal class Property
             internal static readonly int ParentOrigin = Interop.ActorProperty.ParentOriginGet();
             internal static readonly int ParentOriginX = Interop.ActorProperty.ParentOriginXGet();
```

Comment:
Is there any usage with `ParentOriginX` and `ParentOriginY`, instead of `ParentOriginZ`?